tests/integration/targets/s3_object/meta/main.yml

```diff
@@ -1,2 +1,6 @@
 dependencies:
   - setup_remote_tmp_dir
+  # required for s3.get_object_attributes
```

Comment:
It's a lot of effort just to test a botocore function and we may miss some problem with the current support botocore version. I would prefer we stay we the default botocore version and we don't test `object_attributes`.